tools/code_analyzer/build.sh

```diff
@@ -107,9 +107,18 @@ convert_output_to_bazel() {
 
   DEST="${BUILD_ROOT}/pt_deps.bzl"
 
-  python -m tools.code_analyzer.op_deps_processor \
-    --op_dependency "${OUTPUT}" \
+  args=(
+    --op_dependency "${OUTPUT}"
     --output "${DEST}"
+  )
+
+  if [ -n "${BASE_OPS_FILE}" ] && [ -f "${BASE_OPS_FILE}" ]; then
+    args+=(
+      --base_ops $(< ${BASE_OPS_FILE})
```

Comment:
This is just a list of op names, right? Is it worth giving the format anywhere in this code, or is that documented elsewhere/understood by everybody?

Reply:
@bhosmer the format of "base_ops" option is documented in the python script; the bash shell simply cat the file content into the command line.
I'm still working on the integration so this is subject to change. In terminal state the python script will likely be called from some other shell script in fbcode.